pulsar-client/src/main/java/org/apache/pulsar/client/api/CryptoKeyReader.java

```diff
@@ -18,27 +18,36 @@
  */
 package org.apache.pulsar.client.api;
 
+import java.util.List;
+
+import org.apache.pulsar.client.impl.EncryptionKeyInfo;
```

Comment:
If `EncryptionKeyInfo` is part of public API it needs to be in `org.apache.pulsar.client.api` package